docs/source/getting-started/content/proc_starting-the-virtual-machine.adoc

```diff
@@ -8,6 +8,13 @@ The [command]`{bin} start` command starts the {prod} virtual machine and OpenShi
 * The host machine has been set up using the [command]`{bin} setup` command.
 For more information, see <<setting-up-codeready-containers_{context}>>.
 * A valid OpenShift system bundle with the `.crcbundle` extension.
+* A valid OpenShift user pull secret.
+The pull secret can be copied or downloaded from the Pull Secret section of the link:https://cloud.redhat.com/openshift/install/metal/user-provisioned[Install on Bare Metal: User-Provisioned Infrastructure] page on cloud.redhat.com.
```

Comment:
Take note: this is at the moment right, but when our landing page is publshed, tis has to change.

Reply:
Already noted. :)